<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.0.1 (Python 3.12.4 on win32)" generated="2024-09-07T15:55:04.269784" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\jayke\.vscode\extensions\robocorp.robotframework-lsp-1.12.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2024-09-07T15:55:04.293780" elapsed="1.548989"/>
</kw>
<status status="PASS" start="2024-09-07T15:55:04.292782" elapsed="1.549987"/>
</test>
<status status="PASS" start="2024-09-07T15:55:04.271784" elapsed="1.571983"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2024-09-07T15:55:04.265786" level="WARN">Error in file 'c:\Users\jayke\.vscode\extensions\robocorp.robotframework-lsp-1.12.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
